stock_full_location_reservation/models/stock_move.py

```diff
@@ -13,6 +13,22 @@ class StockMove(models.Model):
     def _filter_full_location_reservation_moves(self):
         return self.filtered(lambda m: m.is_full_location_reservation)
 
+    def _action_cancel(self):
+        """
+        Override the method to do the unlink only as the super() do
+        the unreserve also.
```

Comment:
Could we change the doc string to make it more clear? 
```suggestion
        Override the method to avoid the unlink of replenishment moves
        via _do_unsreserve which is getting called in the super
        Instead do the unlink after the super if needed
```